samples/llama_index_quick_start.ipynb

```diff
@@ -0,0 +1,1165 @@
+    "\n",
+    "storage_context = StorageContext.from_defaults(vector_store=vector_store)\n",
+    "index = VectorStoreIndex.from_documents(\n",
+    "    documents, storage_context=storage_context, show_progress=True\n",
+    ")\n",
+    "\n",
+    "# If you have more documents, you can still add them to the database Vector Store table using the embedding service to create embeddings for each record\n",
+    "# vector_store.add_documents(docs_to_load)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Query over indexed data\n",
+    "\n",
+    "A query engine takes in a natural language query and returns a rich response. It is built on one of the indexes, see [Query Engine Guide](https://docs.llamaindex.ai/en/stable/module_guides/deploying/query_engine/).\n",
+    "\n",
+    "You can compose multiple query engines to achieve more advanced querying, see [Query Engine usage patterns](https://docs.llamaindex.ai/en/stable/module_guides/deploying/query_engine/usage_pattern/)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "query_engine = index.as_query_engine()\n",
+    "query = query_engine.query(\"List shows that are about teenagers\")\n",
+    "query.response"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Use case 3: AlloyDB for PostgreSQL as Document Store\n",
+    "\n",
+    "Llama Index parses a document into nodes to store them into Document Store and provides ways to build customized Index for a more sophisticated search. You can also query the `Nodes` of documents from a Document Store."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set up a Document Store"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from llama_index_alloydb_pg import AlloyDBDocumentStore\n",
+    "\n",
+    "document_store_table_name = \"document_store\"\n",
+    "engine.init_doc_store_table(table_name=document_store_table_name)\n",
+    "doc_store = AlloyDBDocumentStore.create_sync(\n",
+    "    engine=engine,\n",
+    "    table_name=document_store_table_name,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Parse documents into nodes\n",
+    "\n",
+    "Using a `TokenTextSplitter`, you can split the movie names on whitespace characters for building a keyword based search index."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from llama_index.core.node_parser import TokenTextSplitter\n",
+    "\n",
+    "splitter = TokenTextSplitter(\n",
+    "    chunk_size=1024,\n",
+    "    chunk_overlap=20,\n",
+    "    separator=\" \",\n",
+    ")\n",
+    "nodes = splitter.get_nodes_from_documents(documents)"
+   ]
```

Comment:
Do we need to parse the docs and add them as nodes to the docstore or can we just set it globally and it will be used with the vector store?

Reply:
Or is this best paired with IndexStore?

Reply:
Following up offline